pkg/operators/klusterlet/controllers/klusterletcontroller/klusterlet_controller.go

```diff
@@ -143,8 +143,15 @@ func NewKlusterletController(
 
 // klusterletConfig is used to render the template of hub manifests
 type klusterletConfig struct {
-	KlusterletName            string
-	KlusterletNamespace       string
+	KlusterletName string
+	// KlusterletNamespace is the klusterlet namespace on the managed cluster.
```

Comment:
Can we add more comments on this to explain something like:
- in default mode, it is identical to the agent namespace.
- in hosted mode, we store some configuration in this namespace that will be used by the agents on the remote management cluster.

Reply:
Doc updated for those two fields.